scripts/remote-dev-context7-device-login.py

```
@@ -0,0 +1,887 @@
+            "license",
+            "dist.integrity",
+            "dist.tarball",
+            "--json",
+        ],
+        uid=uid,
+        gid=gid,
+        cwd=cwd,
+        environment=environment,
+    )
+    if not isinstance(payload, dict):
+        raise DeviceLoginError("official Context7 npm metadata has an unexpected shape")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)